build.gradle

```diff
@@ -281,6 +290,7 @@ subprojects {
         source = "src/main/scala"
         testSource = "src/test/scala"
         failOnWarning = true
+        quiet = true
```

Comment:
This should probably be removed once we figure out how to make the ScalaStyle task pass the Up To Date check.